cognite/client/utils/_auxiliary.py

```diff
@@ -14,6 +14,7 @@
 import string
 import warnings
 from decimal import Decimal
+from ssl import SSLError
```

Comment:
I'm not sure if this is the correct exception. It may be `from requests.exceptions import SSLError`. Try to find a way to reproduce the error and find where the exception stems from.